backend/codesirius/problems/views/problem.py

```
@@ -0,0 +1,132 @@
+            logger.info(f"Problem with ID: {pk} updated successfully")
+            return CodesiriusAPIResponse(
+                data={"id": problem.id}, message="Problem updated"
+            )
+        logger.warning("Problem update failed due to validation errors")
+        raise ValidationError(serializer.errors)
+
+    def patch(self, request: Request, pk: int) -> CodesiriusAPIResponse:
+        """
+        Partially update a problem by its primary key
+        """
+        logger.info(f"Partially updating problem with ID: {pk}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)